hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/service/ContractCallService.java

```
@@ -0,0 +1,63 @@
+package com.hedera.mirror.web3.service;
+
+/*-
+ * ‌
+ * Hedera Mirror Node
+ * ​
+ * Copyright (C) 2019 - 2022 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.INVALID_TRANSACTION;
+
+import javax.inject.Named;
+import lombok.RequiredArgsConstructor;
+import org.apache.tuweni.bytes.Bytes;
+
+import com.hedera.mirror.web3.evm.contracts.execution.MirrorEvmTxProcessorFacade;
+import com.hedera.mirror.web3.exception.InvalidTransactionException;
+import com.hedera.mirror.web3.service.model.CallServiceParameters;
+import com.hedera.node.app.service.evm.contracts.execution.HederaEvmTransactionProcessingResult;
+
+@Named
+@RequiredArgsConstructor
+public class ContractCallService {
+    private final MirrorEvmTxProcessorFacade mirrorEvmTxProcessor;
+
+    public String processCall(final CallServiceParameters body) {
+        final var txnResult = doProcessCall(body);
+        final var callResult = txnResult.getOutput() != null
+                ? txnResult.getOutput() : Bytes.EMPTY;
+
+        return callResult.toHexString();
+    }
+
+    private HederaEvmTransactionProcessingResult doProcessCall(final CallServiceParameters body) {
+        final var processingResult =
+                mirrorEvmTxProcessor.execute(
```

Does this processor catch all exceptions and return a processingResult regardless or should this be wrapped in a try/catch?